RUN_DEPLOY.md

```diff
@@ -0,0 +1,240 @@
+<h1>Document run and deploy app react native</h1>
+
+**This guide builds for Mac OS, does not support Window OS**
+_The running and deploying instructions below are written based on the PIT PORT project, not the sample project._
+
+<h2>Table of Contents</h2>
+
+- [Setup Environment](#setup-environment)
+- [Getting Started](#getting-started)
+- [Getting Started Android](#getting-started-android)
+- [Getting Started iOS](#getting-started-ios)
+- [Project's structures](#projects-structures)
+  - [Introduction](#introduction)
+  - [Main folders, files](#main-folders-files)
+- [Deploy](#deploy)
+  - [Android](#android)
+  - [iOS](#ios)
+
+## Setup Environment
+
+Install the environment (Android, IOS) by following the link: [Setup environment](https://reactnative.dev/docs/environment-setup)
+
+## Getting Started
+
+**The application has 3 environments:**
+
+- **Development**: develop and fix bugs
```

Comment:
**Development**: develop and fix bugs
- Objectives:
  - Coding
  - Fixing bugs
- Audiences:
  - Developers
  - Testers
- Env file: `.env.development`